Jarvis+newengine.py

```diff
@@ -83,13 +83,19 @@ def myCommand():
             speak('Who is the sender? ')
             sender = myCommand()
 
-            if 'I am' in sender:
+            if 'I am' in sender or 'harsh' in sender:
                 try:
                     speak("Please Enter Email address of Recipient.")
                     Recipient_user = input("User: ")
-                    speak('What should I say? ')
-                    content = myCommand()
-
+                    
+                    
+                    speak('What is the subject? ')
+                    subject = myCommand()
+                    
+                    speak('What is the message?')
+                    msg = myCommand()
+                    
+                    content = 'Subject: {}\n\n{}'.format(subject, msg)
```

Comment:
Maybe an overkill but in future we should ask the user if he wants to verify the contents, and repeat it if he says yes.